src/d3d12/d3d12_renderer.hpp

```diff
@@ -96,10 +101,14 @@ namespace wr
 
 		// temporary
 		d3d12::Heap<HeapOptimization::SMALL_BUFFERS>* m_cb_heap;
+		d3d12::Heap<HeapOptimization::BIG_STATIC_BUFFERS>* m_sb_heap;
 
 		d3d12::Viewport m_viewport;
 		d3d12::CommandList* m_direct_cmd_list;
 		d3d12::StagingBuffer* m_fullscreen_quad_vb;
+		d3d12::HeapResource* m_light_buffer;
+
+		Light* m_lights;
```

Comment:
Please don't use C style arrays, prefer std::array and std::vector. C style arrays are relatively error prone.